Services/Administration/OJS.Services.Administration.Business/OJS.Services.Administration.Business.csproj

```diff
@@ -5,6 +5,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\..\..\Servers\Administration\OJS.Servers.Administration.Models\OJS.Servers.Administration.Models.csproj" />
```

Comment:
You should not reference Servers into Services

Reply:
Fixed.